statsmodels/sandbox/distributions/gof_new.py

```diff
@@ -526,8 +526,8 @@ def asquare(cdfvals, axis=0):
     islice = [None] * ndim
     islice[axis] = slice(None)
     slice_reverse[axis] = slice(None, None, -1)
-    asqu = -((2. * np.arange(1., nobs+1)[islice] - 1) *
-            (np.log(cdfvals) + np.log(1-cdfvals[slice_reverse]))/nobs).sum(axis) \
```

Comment:
This is needed to fix a FutureWarning produced by numpy when actually running the code below.